### PR TITLE
Fix tests imports dpr

### PR DESCRIPTION
**Pull request by lhoestq**
There were changes in the locations of some functions used for tests in #5350.
When #5279 was merged some tests couldn't run.

I fixed the imports of those functions.
When I re-ran the tests I noticed that some didn't pass for the DPRReaderTokenizer so I fixed them.

### CODE REVIEW

File: src/transformers/tokenization_dpr.py [hunk 1]
```diff
@@ -203,15 +203,37 @@ class CustomDPRReaderTokenizerMixin:
...
+        texts: Optional[str] = None,
         padding: Union[bool, str] = True,
         truncation: Union[bool, str] = True,
-        max_length: Optional[int] = 512,
+        max_length: Optional[int] = None,
         return_tensors: Optional[Union[str, TensorType]] = None,
         return_attention_mask: Optional[bool] = None,
         **kwargs
     ) -> BatchEncoding:
+        if titles is None and texts is None:
```
Comment (lhoestq) [inline src/transformers/tokenization_dpr.py:215]: I had to handle the cases of having only one or two input strings for the tests.

**Review by LysandreJik — APPROVED** [review]
LGTM, thanks @lhoestq

**Comment by LysandreJik** [comment]
Some tests are failing, please only merge once everything pass.

**Comment by lhoestq** [comment]
Yep I'm on it

**Comment by lhoestq** [comment]
All green, merging

**Pull request merged**